### DISCUSSION TITLE
3rd party support cost (?)

### DISCUSSION THREAD

**Discussion by Data-san** [CMSgov/price-transparency-guide#421] [Q&A] [answered]
Any idea, even roughly, what 3rd party support would cost in building and maintaining the MRFs? Something we might consider. Thanks much.

**Comment by shaselton-usds** [comment]
There are a lot of variables in that question. Probably would help to narrow it a bit.

**Comment by Data-san** [comment]
Thinking about the in-network provider file (one plan), mostly FFS, and the initial build with monthly updates. Is that a six-figure or greater cost per year or something much less? Additional build will depend, in part, how the schema evolves. Just trying to get a rough sense, even a rough range would help. Thanks for the follow-up.

**Comment by shaselton-usds** [comment, marked as answer]
@Data-san It is hard to have any type of estimation without knowing anything about the plans, their contractual arrangements, provider networks, items/services offered, etc.

I would recommend getting a sense of those variables and then building out a small test file, maybe 1/10th or 1/100th the total size and estimate that way. Of course there are many ways to get an estimate too.

**Reply by Data-san** [reply]
Thanks for the added comments and agree, We're making progress with test files.
